src/main/java/seedu/placebook/model/ModelManager.java

```diff
@@ -143,6 +143,7 @@ public boolean hasPerson(Person person) {
     @Override
     public void deletePerson(Person target) {
         addressBook.removePerson(target);
+        this.removePersonFromAppointments(target);
```

Comment:
just checking, the order doesn't matter for these 2 methods cos removePersonFromAppointments doesn't use the addressbook? i just thought maybe there will be a NullPointerException

Reply:
Should not matter, as the addressBook list and appointment list is separate.